internal/repository/prisma/health.go

```diff
@@ -0,0 +1,23 @@
+package prisma
+
+import (
+	"context"
+
+	"github.com/hatchet-dev/hatchet/internal/repository"
+	"github.com/hatchet-dev/hatchet/internal/repository/prisma/db"
+)
+
+type healthRepository struct {
+	client *db.PrismaClient
+}
+
+func NewHealthRepository(client *db.PrismaClient) repository.HealthRepository {
+	return &healthRepository{
+		client: client,
+	}
+}
+
+func (a *healthRepository) IsHealthy() bool {
+	_, err := a.client.User.FindMany().Exec(context.Background())
+	return err == nil
```

Comment:
Figured that this would be the easiest kind of healthcheck for the DB

Reply:
We probably want to limit the number of finds to 1 so we don't run out of memory with thousands of users. We should also check the `dbsqlc` connection here.

Reply:
right good catch

Reply:
done